modules/sandboxed-containers-selecting-nodes.adoc

```diff
@@ -2,68 +2,95 @@
 
 .Prerequisites
 
 * You have {product-title} {product-version} installed on your cluster.
 * You have installed the OpenShift CLI (oc).
 * You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the {sandboxed-containers-operator}.
 
 .Procedure
 
-. Identify the labels that you want to use for selecting your nodes. For this example, use labels to selects to be chosen as candidates to run on your {sandboxed-containers-first} workloads. If the nodes exist, they are selected.
-.. To apply a label to a node, run the following command:
+. Apply a label to a node using the following command:
 +
 [source,terminal]
 ----
-$ oc label node <worker_node_name> <label>=<value>
+$ oc label node <worker_node_name> <label_key=<label_value>
 ----
 +
-This labels your worker node with the `<label>` label that has a value of `<value>`.
-
-. To add a label selector, edit the `KataConfig` custom resource (CR):
-+
+.Example
 [source,terminal]
 ----
-$ oc edit kataconfig
+$ oc label node worker_node_abc custom-kata-machine-pool=true
 ----
-.Example
+
+. Repeat this step for each node you want to label.
+
+. Create a YAML file with the following manifest:
 +
 [source,yaml]
 ----
-  apiVersion: kataconfiguration.openshift.io/v1
-  kind: KataConfig
-  metadata:
-    name: cluster-kataconfig
-  spec:
-    kataConfigPoolSelector:
-      matchLabels:
-         custom-kata-machine-pool: 'true'
+    apiVersion: kataconfiguration.openshift.io/v1
+    kind: KataConfig
+    metadata:
+      name: cluster-kataconfig
+    spec:
+      kataConfigPoolSelector:
+        matchLabels:
+           <label_key>: '<label_value>'
 ----
 
-.Verification
-
-* You can check to see if the nodes in the `machine-config-pool` object are going through a config update.
-** If you are using the default nodes, you can monitor the `machine-config-pool` resource by running:
+. Create the `KataConfig` resource:
 +
 [source,terminal]
 ----
-$ watch oc get mcp worker
+$ oc create -f <file name>.yaml
 ----
-** If you are using selected nodes, you can monitor the `machine-config-pool` resource by running:
+
+The new KataConfig CR is created and begins to install the kata runtime on the worker nodes.
+
+[IMPORTANT]
+====
+{sandboxed-containers-first} installs Kata only as a secondary optional runtime on the cluster and not as the primary runtime.
+====
+
+.Verification
+
+* Monitor the installation progress:
 +
 [source,terminal]
 ----
-$ watch oc get mcp kata-oc
+$ watch oc describe kataconfig
 ----
 
-* You can run `watch oc describe kataconfig cluster-kataconfig` to display information about `sandboxed-containers` extension failure on a node. The information is gathered from the status of the `machine-config-pool` object. You can view the information by running:
 +
-[source,terminal]
-----
-$ oc describe mcp <machine-config-pool>
-----
+Once the value of *Completed nodes* matches the number of labeled nodes, the installation is complete.
```

Comment:
similar comment as before
"Once the number of completed nodes equals the number of labeled nodes..."